web/cobrands/sass/_layout.scss

```diff
@@ -16,6 +16,18 @@ $container-max-width: 60em !default;
 $primary_link_color: null !default;
 $primary_link_hover_color: null !default;
 
+// None of the form variables control the mobile styling. For that refer to _base.scss
+// $front-main-color-desktop controls the colours of all elemets inside #front-main and .postcode-form-box element.
+$front-main-color-desktop: $primary_text !default;
+// $front-main-background-desktop controls the background of the of the #front-main element. This is a full width element.
+$front-main-background-desktop: transparent !default;
+// $front-main-container-desktop changes the background of h1, h2, but without using the full width.
+$front-main-container-background-desktop: $front-main-background-desktop !default;
```

Comment:
Should this be `transparent` or `none`, not front-main-background-desktop? As is, it'll duplicate what's 'underneath' and the padding check in layout will always pass and add padding even when not necessary (e.g. this is happening on Bexley, .com, etc).